pkg/templates/charts/toggle/console-mce/templates/console-plugin.yaml

```diff
@@ -1,22 +1,28 @@
 # Copyright Contributors to the Open Cluster Management project
-apiVersion: console.openshift.io/v1alpha1
+apiVersion: console.openshift.io/v1
 kind: ConsolePlugin
 metadata:
-  name: mce
   annotations:
-    console.openshift.io/use-i18n: "true"
+    installer.multicluster.openshift.io/release-version: 2.7.0
```

Comment:
Do not add this line. We do not want another version that we need to maintain. The installer adds this to the resource when it creates it.